src/Shared/test/Shared.Tests/Microsoft.AspNetCore.Shared.Tests.csproj

```diff
@@ -1,13 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.0;net461</TargetFrameworks>
```

Comment:
Why is this testing net461?

Reply:
@HaoK you updated the .NET Core App version in 2791dbf89278. Did you have a reason to leave the `net461` part alone?

Reply:
No reason, I only noticed the netcoreapp2.2 since it was causing failures on helix

Reply:
:ok: I'm going to delete that TFM as part of this merge.

@Tratcher are the three added references needed in 3.0?

Reply:
Probobly